meilisearch_python_sdk/index.py

```diff
@@ -1288,12 +1288,21 @@ async def search_similar_documents(
 
         return SimilarSearchResults[self.hits_type](**response.json())  # type: ignore[name-defined]
 
-    async def get_document(self, document_id: str) -> JsonDict:
+    async def get_document(
+        self,
+        document_id: str,
+        fields: list[str] | None = None,
+        retrieve_vectors: bool = False,
+    ) -> JsonDict:
```

Comment:
Lets make these name only.

```suggestion
    async def get_document(
        self,
        document_id: str,
        *,
        fields: list[str] | None = None,
        retrieve_vectors: bool = False,
    ) -> JsonDict:
```